api/src/services/graphql/schema-cache.ts

```diff
@@ -0,0 +1,16 @@
+import { useEnv } from '@directus/env';
+import { GraphQLSchema } from 'graphql';
+import LRUMapDefault from 'mnemonist/lru-map.js';
```

Comment:
Could we use `@directus/memory` instead? 🤔

Reply:
That doesn't implement an LRU system, so no! That's just regular k/v through Redis, rather than a local LRU map 👍🏻